templates/ingress.yaml

```diff
@@ -16,19 +17,14 @@ metadata:
 {{ toYaml . | indent 4 }}
 {{- end }}
   annotations:
-    kubernetes.io/ingress.class: {{ .Values.ingress.class }}
-    kubernetes.io/tls-acme: {{ eq .Values.ingress.acme true | quote }}
```

Comment:
Likewise here -- removing this annotation makes the `Values.ingress.acme` unused by default.